RealmSwift/Impl/Persistable.swift

```diff
@@ -57,6 +60,10 @@ extension _Persistable {
     }
 }
 
+public protocol _BuiltInPersistable {
```

Comment:
Would be nice to have a comment explaining what this is. I can assume what `BuiltIn` means in this context, but it would nice to be sure.

Reply:
Added a short note.